.env.example

```diff
@@ -1,7 +1,6 @@
-DISCORD_TOKEN=
-DISCORD_PUBKEY=
-DATABASE_URL=
-REDIS_URL=
-ROOT_URL=
-CONTROL_GUILD=
-OWNERS=
\ No newline at end of file
+POSTGRES_PASSWORD=<db_pass>
+DATABASE_URL=postgres://xpd:$(POSTGRES_PASSWORD)@postgres:5432/xpd
```

Comment:
the env won't fill itself, you have to do this in the compose file

Reply:
I tested it locally and it worked-

Reply:
*what*
if so, that's news to me. i just don't see a good reason in keeping this in the .env, when none of the content is actually relevant to the user